From 4b07ec34baca9ac07b1e9814dc0f06c1662f8dd2 Mon Sep 17 00:00:00 2001
From: "sachin.shetty" <sachin.shetty@lge.com>
Date: Mon, 24 Jun 2019 16:55:43 +0530
Subject: [PATCH] Multiple wi-fi networks are connected via WPS PIN.

:Release Notes:
Fix multiple services in associated

:Detailed Notes:
It undo 'associated network' changes added in connman#1.35.
Connman commit ids
6b925ee3baec13ab7fe31453278fe515f04d9c21
f70611bf6ac92dc7be19cfe150054a2994371715
464ae52378f2bb7d35451a554583569feea8f767

:Testing Performed:
Build is successfully done.
Verified the image.

:QA Notes:

:Issues Addressed:
[PLAT-80478] [webOS OSE]: Multiple wi-fi networks are
             connected via WPS PIN.

Upstream-Status: Inappropriate [webOS specific]
---
 gsupplicant/supplicant.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gsupplicant/supplicant.c b/gsupplicant/supplicant.c
index 1a35d00a..a8460783 100644
--- a/gsupplicant/supplicant.c
+++ b/gsupplicant/supplicant.c
@@ -2370,7 +2370,7 @@ static void interface_property(const char *key, DBusMessageIter *iter,
 				g_strdup(interface->ifname), g_strdup(str));
 		}
 	} else if (g_strcmp0(key, "CurrentBSS") == 0) {
-		interface_current_bss(interface, iter);
+		interface_bss_added_without_keys(iter, interface);
 	} else if (g_strcmp0(key, "CurrentNetwork") == 0) {
 		interface_network_added(iter, interface);
 	} else if (g_strcmp0(key, "BSSs") == 0) {
